src/routes/pass/[id]/+page.svelte

```diff
@@ -45,12 +45,12 @@
 
 <div
 	id="my-pass-container"
-	class="mx-4 space-y-6 pb-36 pt-4"
+	class="mx-4 min-h-[calc(100vh-64px)] space-y-6 pb-2 pt-4"
```

Comment:
is there a cleaner way to pull/inject the top-level `+layout` bottom nav height? this only works because we have it in this mobile-only bottom layout at the moment